include/coreir-c/coreir-values.h

```diff
@@ -16,6 +16,7 @@ extern COREValue* COREValueInt(COREContext* c,int i);
 extern COREValue* COREValueString(COREContext* c,char* str);
 extern COREValue* COREValueBitVector(COREContext* c, int width, uint64_t val);
 extern COREValue* COREValueModule(COREContext* cc, COREModule* mod);
+extern COREValue* COREValueCoreIRType(COREContext* cc, COREType* mod);
```

Comment:
Can you rename `mod` -> `type` to match the implementation (and make the parameter name consistent with the function name)

Reply:
Change has been made.